plugins/CoreHome/templates/_headerMessage.twig

```diff
@@ -29,14 +29,18 @@
 
     <div class="dropdown">
         {% if isPiwikDemo %}
-            {{ 'General_DownloadFullVersion'|translate("<a href='http://piwik.org/'>","</a>","<a href='http://piwik.org'>piwik.org</a>")|raw }}
+            {{ 'General_DownloadFullVersion'|translate("<a rel='noreferrer' href='https://piwik.org/'>","</a>","<a rel='noreferrer' href='https://piwik.org'>piwik.org</a>")|raw }}
             <br/>
             {% if isSuperUser and adminMenu is defined and adminMenu %}
                 <br/>
             {% endif %}
         {% endif %}
         {% if latest_version_available and isSuperUser %}
-            {{ 'General_PiwikXIsAvailablePleaseUpdateNow'|translate(latest_version_available,"<br /><a href='index.php?module=CoreUpdater&amp;action=newVersionAvailable'>","</a>","<a href='?module=Proxy&amp;action=redirect&amp;url=http://piwik.org/changelog/' target='_blank'>","</a>")|raw }}
+            {% if isMultiServerEnvironment %}
+                {{ 'CoreHome_OneClickUpdateNotPossibleAsMultiServerEnvironment'|translate("<a rel='noreferrer' href='https://builds.piwik.org/piwik-" ~ latest_version_available ~ ".zip'>","</a>")|raw }}
```

Comment:
Instead of just downloading latest zip we will point to exact version since there might be Piwik 3.0.0 updates etc. Version and link to zip basically depends on release channel